docx_test/resources/package.go

```diff
@@ -0,0 +1 @@
+package resources
```

Comment:
If you instead use a `testdata` directory then you won't need to declare it as a pkg:

```shell
$ go help test
...
The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.
...
```

Reply:
Interesting, thank you for sharing this info. Done.